<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialog</class>
 <widget class="QDialog" name="Dialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>260</width>
    <height>130</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QWidget" name="button_widget" native="true">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>85</y>
     <width>240</width>
     <height>40</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="button_layout">
    <item>
     <widget class="QPushButton" name="confirm_button">
      <property name="text">
       <string>Confirm</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="cancel_button">
      <property name="text">
       <string>Cancel</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>10</y>
     <width>151</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>&lt;b&gt;Select the SD Card Drive:&lt;/b&gt;</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
